<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSA Assignment 1</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="labelPK_e">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Public Key (e)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textPublicKey_e">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textPrivateKey">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPRK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Private Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Private &amp; Public Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEncMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt / Decrypt Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEncFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt / Decrypt File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textPublicKey_n">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBK_n">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Public Key (n)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
